app/controllers/unattended_controller.rb

```diff
@@ -76,7 +80,7 @@ def preview?
     params.key?(:spoof) || params.key?(:hostname)
   end
 
-  def render_custom_error(status, error_message, params)
+  def render_error(status, error_message, params)
```

Comment:
This method is defined in the concern and redefined here, is this intentional?

Reply:
Yeah, we need different behavior.